dvc/data_cloud.py

```diff
@@ -64,7 +64,7 @@ def _init_odb(self, name):
         return get_odb(cls(**config), fs_path, **config)
 
     def _log_missing(self, status: "CompareStatusResult"):
-        if status.missing:
+        if status.missing and logger.isEnabledFor(logging.WARNING):
```

Comment:
hm, could you explain the reasoning behind this change, please? Seems unrelated to the rest.

Reply:
@alexmojaki That seems unnecessary though. I'll adjust it back for now